EKF/estimator_interface.cpp

```diff
@@ -62,22 +62,31 @@ EstimatorInterface::EstimatorInterface():
 	_yaw_test_ratio(0.0f),
 	_tas_test_ratio(0.0f),
 	_terr_test_ratio(0.0f),
+	_beta_test_ratio(0.0f),
 	_time_last_imu(0),
 	_time_last_gps(0),
 	_time_last_mag(0),
 	_time_last_baro(0),
 	_time_last_range(0),
 	_time_last_airspeed(0),
 	_time_last_ext_vision(0),
+	_time_last_optflow(0),
 	_mag_declination_gps(0.0f),
-	_mag_declination_to_save_deg(0.0f)
+	_mag_declination_to_save_deg(0.0f),
+	_pos_ref{},
+	_mag_test_ratio{},
+	_vel_pos_test_ratio{},
+	_vibe_metrics{},
+	_mag_sample_delayed{},
```

Comment:
@bkueng Can you tell me if this kind of initialisation works for structs with non POD members?